docs/requirements.txt

```diff
@@ -6,9 +6,9 @@ scikit-learn
 sphinx>=4.0,<5.0
 sphinx_rtd_theme>=1.0
 tensorflow
-transformers>=4.6.*
-torch==1.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
-pytorch-lightning==1.6.5
+transformers==4.34.1
```

Comment:
Let's update transformers to track the latest dependency

Reply:
Let's make this a lower bound as well

Reply:
done

Reply:
This still needs to be bumped. Can you put up a small fix PR? I'll merge this in since it's fixed in the actual files below